package.yaml

```diff
@@ -16,21 +16,22 @@ author: Justus Adam
 maintainer: dev@justus.science
 copyright: (c) 2015 - 2022 Justus Adam
 license: BSD3
-homepage: https://github.com/JustusAdam/mustache
-git: git://github.com/JustusAdam/mustache.git
-bug-reports: https://github.com/JustusAdam/mustache/issues
-tested-with: GHC>=7.8 && <=7.10.2
+
+github: JustusAdam/mustache
```

Comment:
This is functionally equivalent to specifying`homepage`, `bug-reports`, and `git` individually.